test/functional/tests/cli/test_cli_start_stop.py

```diff
@@ -7,6 +7,8 @@
 import pytest
 
 from api.cas import casadm, casadm_parser
+from api.cas.cli import start_cmd
+from api.cas import cli_messages
```

Comment:
You can add cli_messages to previous modules imported from api.cas (two lines above) just for cleanliness.